repl/src/main/scala/com/cloudera/livy/repl/SparkRInterpreter.scala

```diff
@@ -125,9 +126,10 @@ object SparkRInterpreter {
   }
 }
 
-class SparkRInterpreter(process: Process, backendInstance: Any, backendThread: Thread)
-  extends ProcessInterpreter(process)
-{
+class SparkRInterpreter(process: Process,
+                        backendInstance: Any,
```

Comment:
Personally I have no opinion on this. But existing code follows Spark's coding style. Let's be consistent and follow Spark's coding style.